recipes/transposonpsi/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set name = "transposonpsi" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "9ea02dc62f1a37137d0b458e41d3832baf25d83c06471e3150699d8a515236df" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: {{ version }}
+
+build:
+  noarch: generic
+  number: 0
+
+source:
+  url: https://github.com/NBISweden/TransposonPSI/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+requirements:
+  host:
+    - perl-bioperl >=1.7.2
+    - blast-legacy=2.2.26
```

Comment:
Please add `- perl` as a dependency

Reply:
...both here and in the `run:` section

Reply:
Ok I thought as Bioperl requires anyway perl It was not necessary to mention it. I will add it